util/apicaps/caps.go

```diff
@@ -0,0 +1,161 @@
+package apicaps
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	pb "github.com/moby/buildkit/util/apicaps/pb"
+	"github.com/pkg/errors"
+)
+
+type PBCap = pb.APICap
+
+// ExportedProduct is the name of the product using this package.
+// Users vendoring this library may override it to provide better versioning hints
+// for their users (or set it with a flag to buildkitd).
+var ExportedProduct string = "buildkit"
+
+// CapStatus defines the stability properties of a capability
+type CapStatus int
+
+const (
+	// CapStatusStable should never be changed in backwards incompatible manner unless
+	// there is a serious security issue.
+	CapStatusStable CapStatus = iota
+	// CapStatusExperimental may be removed in the future. If incompatible changes are
```

Comment:
`CapStatusExperimental refers to a capability that may be removed in the future.`